docs/_docs/Starters Academy (LDSSA)/05-Capstone-Project.md

```diff
@@ -81,15 +93,19 @@ In order to pass the capstone all the following must be delivered and pass a set
 
 #### API
 
-All students should have delivered a working API by report 1 to pass. This will be verified by the following:
+All students should have delivered a working API by report 1 to pass. This will be verified by the 
+following:
 
 * the API responds to the requests - anything other than a 404;
 * an intermediate delivery of the app code may be requested for confirmation.
 
 
 #### Report
 
-In order to get a passing chance, both reports need to be sent with **all sections**. In addition, each section has to pass a minimum threshold of quality, with respect to the requirements presented. You can refer to the [sections and descriptions of batch 4](TBD) to understand better how these may look.
+In order to get a passing chance, both reports need to be sent with **all sections**. In addition, 
```

Comment:
"with **all sections**" -> "with **all sections completed**."
Just to totally avoid any trace of ambiguity. 😅

Reply:
Fixed